pkg/types/upgrades.go

```diff
@@ -0,0 +1,11 @@
+package types
+
+type ChangeReport struct {
+	ComponentReports []ComponentChangeReport
+}
+
+type ComponentChangeReport struct {
```

Comment:
Report makes me think of a list of information in a text based format. What about `ComponentChangeDiff`, `ComponentDisparity`, or `ComponentCompare`

Reply:
I like `ComponentDiff`, will change

Reply:
Done